package.json

```diff
@@ -31,7 +31,7 @@
     "cross-spawn": "7.0.1",
     "fs-extra": "9.0.0",
     "generator-jhipster": ">=6.5.1",
-    "jhipster-core": "7.1.1",
+    "jhipster-core": "^7.1.1",
```

Comment:
Might as well keep it as it is, no?

Reply:
The commit description:
```
Allow jhipster-core to update not majors versions.
generator-jhipster is not pinned, so pin jhipster-core to version 7
instead of 7.1.1.
```

Since generator-jhipster is not pinned, then I think jhipster-core shouldn't be pinned.
If generator-jhipster 6.8.1 uses jhipster-core 7.2.0, I think it's better to let jhipster-core update to 7.2.0:
The backward compatibility should be better then the forward compatibility.

Reply:
That's not a necessary change, that's why I commented. I'm completely neutral about this.

Reply:
Not necessary.
Let the maintainer decide.

Reply:
Do you think I should specify jhipster-core as a dependency, or just get it through `generator-jhipster`? I was just trying to make things work when I added it yesterday.

Reply:
I think you should.
If you remove and then run with generator-jhipster@6.7.0 (jhipster-core@6.0.7) it will fail.